system/netdata.service.in

```diff
@@ -12,7 +12,9 @@ Group=netdata
 RuntimeDirectory=netdata
 RuntimeDirectoryMode=0775
 PIDFile=@localstatedir_POST@/run/netdata/netdata.pid
-ExecStart=@sbindir_POST@/netdata -P @localstatedir_POST@/run/netdata/netdata.pid -D -W set global 'process scheduling policy' 'keep' -W set global 'OOM score' 'keep'
+Environment=DAEMONOPTS="-P @localstatedir_POST@/run/netdata/netdata.pid"
```

Comment:
The quotes here should include the full assignment as per the documentation
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Environment=

It should be:
```
Environment="DAEMONOPTS=-P @localstatedir_POST@/run/netdata/netdata.pid"
```

Reply:
If you test the branch you will notice that the service is not started with the full list of parameters. something a bit unexplainable happens, but when you change it to the above it passes all parameters correctly

Reply:
i finally connected and changed it (i've been out last days)